scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
+
+    Returns
+    -------
+    value : float
+        Value of the test statistic
+
+    Notes
+    ------
+    Cramer's V and Tschuprow's T measure degree to which two variables are
+    related, or the level of their association. This differs from correlation,
+    although many often mistakenly consider them equivalent.
+    Correlation measures in what way two variables are related, whereas,
+    association measures how related the variables are. As such, association
+    does not subsume independent variables, and is rather a
+    test of independence. A value of 1.0 indicates perfect association, and
+    0.0 means the variables have no association.
+
+    Both the Cramer's V and Tschuprow's T are extensions of the phi coefficient.
+    Moreover, due to the close relationship between the Cramer's V and
+    Tschuprow's T the returned values can often be similar or even equivalent.
+    They are likely to diverge more as the array shape diverges from a 2x2.
+
+    The evaluation of Pearsons Contingency Coefficient is not effected by the
+    bias correction metric, because it was not included as a
+    part of the supporting academic paper.
+
+    References
+    ----------
+    .. [1] "Tschuprow's T",
+           https://en.wikipedia.org/wiki/Tschuprow's_T
+    .. [2] Tschuprow, A. A. (1939)
+           Principles of the Mathematical Theory of Correlation;
+           translated by M. Kantorowitsch. W. Hodge & Co.
+    .. [3] "Cramer's V", https://en.wikipedia.org/wiki/Cramer's_V
+    .. [4] "Nominal Association: Phi and Cramer's V",
+           http://www.people.vcu.edu/~pdattalo/702SuppRead/MeasAssoc/NominalAssoc.html
+    .. [5] Gingrich, Paul, "Association Between Variables",
+           http://uregina.ca/~gingrich/ch11a.pdf
+
+
+    Examples
+    --------
+    2-way Example
+
+    >>> from scipy.stats.contingency import association, chi2_contingency
+    >>> obs = np.array([[100, 150], [203, 322], [42, 7], [32, 21]])
+    >>> chi2 = chi2_contingency(obs)[0]
+
+    Pearson's contingency coefficient
+    
+    >>> association(obs, chi2, stat="C")
+    0.22768
+
+    Cramer's V
+
+    >>> association(obs, chi2, stat="V")
+    0.23382
+
+    Tschuprow's T
+
+    >>> association(obs, chi2, stat="T")
+    0.17766
+
+    Phi
+
+    >>> association(obs, chi2, stat="phi")
+    0.23382
+    """
+    arr = np.asarray(arr)
+
+    if stat.lower() not in ['tschuprow', "pearson", 'cramer', 'phi',
+                            't', 'c', 'v']:
+        raise ValueError("stat must be in "
+                         "['tschuprow', 'cramer', 'pearson', 'phi']")
+    else:
+        stat_str = stat.lower()
+
+    if len(arr.shape) != 2:
+        raise ValueError("method only accepts 2d arrays")
+
+    else:
+        try:
+            chi2_stat = float(chi2_stat)
+        except ValueError:
+            raise ValueError("Invalid chi2_stat, must be float or int")
+        else:
+            phi2 = chi2_stat / sum(arr.flatten())
```

Comment:
Just notice one more change...
```suggestion
            phi2 = chi2_stat / arr.sum()
```